charmcraft.yaml

```
@@ -16,6 +16,10 @@ parts:
   charm:
     build-packages:
     - "git"
+    charm-binary-python-packages:
+      # If we build pyyaml from source, then we won't have the C-optimized lib.
```

```suggestion
      # If we build pyyaml from source, then we will have the C-optimized lib.
```
You meant we will have it, correct?